README.md

```diff
@@ -90,9 +88,9 @@ Below you can find the information on the custom options to set up the configura
 | _sampleRate_ | Defines the rate sampling. **It should be a number between [1, 100]**. | `number` | `100` | **NO** |
 | _tags_ | Object for setting the global tags. | `object` | `{}` | **NO** |
 | _transport_ | Defines the transport layer to be used to send metrics.<br><br> **Valid Transports:** `udp, api` | `string` | **none** | **YES** |
-| _host_ | Defines the hostname to where the metrics are sent. It can only be set inside _api_. | `string` | `127.0.0.1` | **NO** |
+| _host_ | Defines the hostname to where the metrics are sent. It can be set for _api_ or _udp_ whichever is applicable. | `string` | udp:   `127.0.0.1`<br>api: `'api.statful.com'` | **NO** |
```

Comment:
"It can be set for _api_ or _udp_ whichever is applicable" this isn't correct this is realted to "transport" attribute.

Here it should be "Defines the hostname to where the metrics are sent. It can only be set inside _api_. Default values vary depending on _transport_ value."